docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -113,7 +113,7 @@ Object colors can vary dramatically based on the light source.
 We recommend you verify the desired color detection value under actual lighting conditions.
 To determine the color value from the actual cam component image, you can use a pixel color tool, like [Color Picker for Chrome](https://chrome.google.com/webstore/detail/color-picker-for-chrome/clldacgmdnnanihiibdgemajcfkmfhia).
 
-If the color is not reliably detected, increase the `hue_tolerance_pct`.
+If the color is not reliably detected, drag the `Hue Tolerance` slider to the right or navigate to the `Raw JSON` tab to increase the `hue_tolerance_pct`.
```

Comment:
```suggestion
If the color is not reliably detected, drag the **Hue Tolerance** slider to the right or navigate to the `Raw JSON` tab to increase the `hue_tolerance_pct`.
```
Since in this case "Hue Tolerance" isn't a string that is part of the code or the JSON, we don't use monospace. If it was "hue_tolerance" you would, because that'd be the exact string and would be referring to the JSON instead of to the UI.